app/views/articles/show.html.erb

```diff
@@ -103,73 +103,64 @@
+                  <% end %>
+                  <p class="fs-xs color-base-60">
+                    <% if @article.published_timestamp.present? %>
+                      Posted on
+                      <%= local_date(@article.published_timestamp, show_year: @article.displayable_published_at.year != Time.current.year) %>
+                    <% end %>
+                    <% if @article.co_author_ids.present? %>
+                      with <%= @article.co_author_name_and_path.html_safe %>
+                    <% end %>
+
+                    <% if should_show_updated_on?(@article) %>
+                      &bull; Updated on <%= local_date(@article.edited_at, show_year: @article.edited_at.year != Time.current.year) %>
+                    <% end %>
+
+                    <% if should_show_crossposted_on?(@article) %>
+                      &bull; Originally published at
+                      <a href="<%= @article.canonical_url || @article.feed_source_url %>" style="color:#1395b8"><%= get_host_without_www(@article.canonical_url || @article.feed_source_url) %></a>
+                      <% if @article.crossposted_at %>
+                        on
+                        <%= local_date(@article.originally_published_at || @article.published_at, show_year: (@article.originally_published_at || @article.published_at).year != Time.current.year) %>
+                      <% end %>
+                    <% end %>
+                  </p>
+                </div>
+              </div>
+            </div>
+
+            <h1 class="fs-3xl m:fs-4xl l:fs-5xl fw-bold s:fw-heavy lh-tight mb-2 <%= @article.title_length_classification %>">
               <% if @article.search_optimized_title_preamble.present? && !user_signed_in? %>
                 <span class="fs-xl color-base-70 block"><%= @article.search_optimized_title_preamble %></span>
               <% end %>
               <%= @article.title %>
             </h1>
 
             <% cache("main-article-tags-#{@article.cached_tag_list}", expires_in: 30.hours) do %>
-              <div class="mb-4 spec__tags">
+              <div class="spec__tags color-base-60">
                 <% @article.cached_tag_list_array.each do |tag| %>
-                  <% tag_hash = tag_colors(tag) %>
-                  <% styles = "" %>
-                  <% if tag_hash[:background] %>
-                    <% styles = "background-color:#{tag_hash[:background]};" %>
-                  <% end %>
-                  <% if tag_hash[:color] %>
-                    <% styles = "#{styles}color:#{tag_hash[:color]};" %>
-                  <% end %>
-                  <a class="crayons-tag mr-1" href="/t/<%= tag %>" style="<%= styles %>"><span class="crayons-tag__prefix">#</span><%= tag %></a>
+                  <a href="/t/<%= tag %>" class="crayons-link crayons-link--secondary"><%= tag %></a><%= "," unless tag == @article.cached_tag_list_array.last %>
```

Comment:
nitpick: prefixing tags with `#` makes it more apparent they are tags. It does not seem obvious that they are links because it is a comma separated list. Are there hover styles for these tags?

Reply:
It's a `crayons-link` so there is definitely a hover state for these (at least should be 😅).

Re `#` - By removing this I'm trying to persuade the idea of "topics" - this was the term I've been hearing quite often recently in context of Forem. So I'd say I've done it intentionally but at the same time this is not the hill I'm ready to die on :)

Reply:
It's not a hill I want to die on either lol. I'll let some other folks chime in.

Reply:
seems like people have mixed opinions about it - i'm reverting this change.

Reply:
I'm looking at it again, and I'm not minding it without the `#` now. I think it's just getting used to the change is all. 🙃